charts/enterprise-logs/multi-kv-memberlist-primary-values.yaml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
We won't need this file which exists to provide a migratory path between Consul and memberlist as ring KV stores.